causing/estimate.py

```diff
@@ -526,23 +526,31 @@ def estimate_biases(model_dat):
 def estimate_models(model_dat):
     """estimation of modification indicators of level model"""
 
+    xmean = model_dat["xdat"].mean(axis=1)
+    xcdat = model_dat["xdat"] - xmean.reshape(model_dat["mdim"], 1)
+    ymmean = model_dat["ymdat"].mean(axis=1)
+    ymcdat = model_dat["ymdat"] - ymmean.reshape(model_dat["pdim"], 1)
+
     selvec = zeros(model_dat["ndim"])
     selvec[[list(model_dat["yvars"]).index(el) for el in model_dat["ymvars"]]] = 1
     selmat = diag(selvec)
     selvec = diag(selmat)
     selvec = diag(selmat)
```

Comment:
this line is redundant and selvec is already known from two lines before

Reply:
This is exactly what happened before in the code, it was just spread out a bit more. But I'm happy to remove it.

Reply:
Yeah, I know it already happened before, but now it became quite obvious that it could be improved upon :)